boards/wch/linkw/linkw.dts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2024 MASSDRIVER EI (massdriver.space)
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <wch/ch32v208.dtsi>
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include "linkw-pinctrl.dtsi"
+
+/ {
+	model = "linkw";
+	compatible = "wch,ch32v208";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &usart2;
+		zephyr,shell-uart = &usart2;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		status = "okay";
+
+		green_led: led0 {
+			gpios = <&gpioc 7 GPIO_ACTIVE_LOW>;
+		};
+		blue_led: led1 {
```

Comment:
inconsistent spacing

Reply:
Does this latest modification fixes this?

Reply:
As I understand it, he meant to insert empty lines around blocks like it is done [here](https://github.com/zephyrproject-rtos/zephyr/pull/82704/files#diff-8113cd017ce1f612f953d129bf81d0a4c19213fcfeddc38011000b95127dbc14R73-R119) for instance (feel free to correct me if I misinterpreted).

Reply:
Done.